hal/halx86/apic/rtctimer.c

```diff
@@ -188,6 +191,9 @@ HalpClockInterruptHandler(IN PKTRAP_FRAME TrapFrame)
 
     /* Update the system time -- on x86 the kernel will exit this trap  */
     KeUpdateSystemTime(TrapFrame, LastIncrement, Irql);
+
+    /* End the interrupt */
```

Comment:
I don't know, maybe this could be surrounded with `#ifndef _M_IX86 ... #endif` , since it would be necessary only on non-x86.